tests/phpunit/unit/MediaWiki/Request/EditRequestParserTest.php

```diff
@@ -122,14 +123,14 @@ public function testParseRequestInterface_good(): void {
 	}
```

Comment:
There should probably be an assertion here about the `$editRequest->token()`? (Unless you want to squash-merge the PR, in which case it’s moot because the `token()` goes away again anyways ^^)